### DISCUSSION TITLE
Env for OLLAMA_API_BASE_URL not properly saving

### DISCUSSION THREAD

**Discussion by PokeFoundry** [open-webui/open-webui#2177] [General]
I am running Open WebUI on Unraid, here is the docker run command:

`docker run
  -d
  --name='open-webui'
  --net='bridge'
  -e TZ="America/Los_Angeles"
  -e HOST_OS="Unraid"
  -e HOST_HOSTNAME="Tower"
  -e HOST_CONTAINERNAME="open-webui"
  -e 'OLLAMA_API_BASE_URL'='http://10.10.1.201:11434/api'
  -e 'OLLAMA_ORIGINS'='*'
  -l net.unraid.docker.managed=dockerman
  -l net.unraid.docker.webui='http://[IP]:[PORT:8080]/'
  -l net.unraid.docker.icon='https://raw.githubusercontent.com/open-webui/open-webui/main/static/favicon.png'
  -p '1337:8080/tcp'
  -v '/mnt/user/system/docker/ollama-webui':'/app/backend/data':'rw' 'ghcr.io/open-webui/open-webui:main'
65b766cbd270ac44987c893b2f1ef90e9af75cd0eaac074967574dbe9a8b19be

The command finished successfully!`

Issue I am having is `ERROR:apps.ollama.main:Connection error: Cannot connect to host host.docker.internal:11434 ssl:default [Name or service not known]`

While changing this value in the settings works, the next time the container is restarted, the settings are gone and the same issue re-appears.

**Comment by justinh-rahb** [comment]
Outdated.

**Reply by PokeFoundry** [reply]
I'm sorry could you elaborate further. Is the docker outdated or is there something else I am missing? Thank you. :)
